com.unity.render-pipelines.universal/Shaders/Lit.shader

```diff
@@ -199,6 +207,7 @@ Shader "Universal Render Pipeline/Lit"
             #pragma shader_feature_local_fragment _METALLICSPECGLOSSMAP
             #pragma shader_feature_local_fragment _SMOOTHNESS_TEXTURE_ALBEDO_CHANNEL_A
             #pragma shader_feature_local_fragment _OCCLUSIONMAP
+            // SM2.0: NOT SUPPORTED shader_feature_local _DETAIL_MULX2
```

Comment:
This information should be added to docs.

Reply:
Fixed